test-e2e/tests/seo_test.ts

```diff
@@ -86,7 +85,7 @@ Scenario('It renders the correct structured metadata for the series screen', asy
       uploadDate: '2021-03-10T10:00:00.000Z',
       partOfSeries: {
         '@type': 'TVSeries',
-        '@id': `${constants.baseUrl}s/${seriesId}`,
+        '@id': `${constants.baseUrl}m/oXGyKQ97`,
```

Comment:
The reason this was written dynamically was so that the tests were more resilient to data changes. I made an effort to take all of the hard coded IDs out of the e2e tests so that if an item gets deleted and recreated for example, the tests don't need updates and they can be run on another property as long as the content is copied.

Reply:
Fixed